### PR TITLE
[TS] LPS-194909: A journal article cannot be re-indexed after deleting fields within a field set, itself contained in a fields grouped, in a web content structure

### PR DESCRIPTION
**Pull request by jvelalr**
Can you review these changes in order solve [LPS-194909](https://liferay.atlassian.net/browse/LPS-194909) issues?
Thanks you.

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java [hunk 1]
```diff
@@ -805,6 +805,10 @@ private void _extractIndexableAttributes(
 		for (DDMFormFieldValue ddmFormFieldValue : ddmFormFieldValues) {
 			DDMFormField ddmFormField = ddmFormFieldValue.getDDMFormField();
 
+			if (ddmFormField == null) {
+				return;
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java:809]: @jvelalr please change to `continue;` instead

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java (hunk 1 above)
Comment (jvelalr) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java]: @carolmariaabb done!, thanks.

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:reevaluate:1257401_3023

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**